lib/datadog/tracing/contrib/ethon/easy_patch.rb

```diff
@@ -139,7 +142,9 @@ def datadog_tag_request
               span.set_tag(Tracing::Metadata::Ext::NET::TAG_TARGET_HOST, uri.host)
               span.set_tag(Tracing::Metadata::Ext::NET::TAG_TARGET_PORT, uri.port)
 
-              span.set_tag(Tracing::Metadata::Ext::TAG_PEER_HOSTNAME, uri.host)
+              if Contrib::SpanAttributeSchema.default_span_attribute_schema?
+                span.set_tag(Tracing::Metadata::Ext::TAG_PEER_HOSTNAME, uri.host)
```

Comment:
The other integrations are not guarded by the conditions, any reason for this?

Reply:
Fixed